templates/about/release-cycle.html

```diff
@@ -224,8 +224,8 @@ <h2>Maintenance and security updates</h2>
             </tr>
             <tr>
               <td><strong>universe</strong></td>
-              <td>ESM</td>
-              <td>ESM</td>
+              <td><a class="js-invoke-modal" href="https://www.ubuntu.com/contact-us/form/">Contact us&nbsp;&rsaquo;</a></td>
```

Comment:
These links don't work for me on [the demo](https://ubuntu-com-canonical-web-and-design-pr-7586.run.demo.haus/about/release-cycle)

Reply:
there is an issue with the JS, so I made them plain for now... can you re-test?